src/figure.rs

```diff
@@ -230,10 +230,56 @@ impl Figure
 		self
 	}
 
-	/// Closes the gnuplot process.
-	///
-	/// This can be useful if you're your plot output is a file and you need to
-	/// that it was written.
+	/// Save the figure to a png file
+	/// # Arguments
+	/// * `file_path` - Path to the output file (png)
+	pub fn save_to_png(&mut self, file_path: &str)
```

Comment:
Could you expose a few common arguments for all of these? At the very least, I think it makes sense to expose the output width and height. Be careful though, since the units are different for different terminals, I'd probably call the arguments `width_px` for pixels, `width_in` for inches etc.

Reply:
Do you mean the figure width (and height) ?
I can't find any mention of those values in code.
However parsing Gnuplot's output in order to get Figure's attribute and/or errors would be a great.

Reply:
No, what I mean is that for many (all?) terminals, you can set the size of the figure. For example, you can do `set terminal pngcairo size 256,256`. In the code here, this could mean `format!("png cairo size {},{}", width_px, height_px)`

Reply:
Ok sorry for the confusion, I am working on it